lib/sign_dict_web/controllers/entry_controller.ex

```diff
@@ -51,7 +51,7 @@ defmodule SignDictWeb.EntryController do
   def new(conn, _params) do
     changeset = Entry.changeset(%Entry{})
     languages = Repo.all(Language)
-    render(conn, "new.html", changeset: changeset, languages: languages)
+    render(conn, "new.html", changeset: changeset, languages: languages, text: conn.params["text"])
```

Comment:
Maybe you could rename the `_params` to `params` and use that to get the text? Would that be possible?